### PR TITLE
Moved week 7,8,9 from the JavaScript repo to this repository (So JavaScript3). 

### PR DESCRIPTION
**Pull request by mkruijt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+# JavaScript3
 
 Here you can find course content and homework for the JavaScript3 modules
 
 |Week|Topic|Read|Homework|
 |----|-----|----|--------|
-|1.|• [Object Oriented Programming and Classes](fundamentals/oop_classes.md)<br>• [The `this` keyword](fundamentals/this.md) |[Reading Week 7](/Week1/README.md)|[Homework Week 7](/Week1/MAKEME.md)|
-|2.|• Structure for a basic SPA (Single Page Application) <br>• [XMLHttpRequests](fundamentals/XMLHttpRequest.md) <br>• API calls|[Reading Week 8](/Week2/README.md)|[Homework Week 8](/Week2/MAKEME.md)|
-|3.|• [Promises](fundamentals/promises.md)<br> • (re)writing data structures (in JSON) <br>• Async vs Sync <br>• [Event Loop (order of execution)](fundamentals/event_loop.md) |[Reading Week 9](/Week3/README.md)|[Homework Week 9](/Week3/MAKEME.md)|
+|1.|• [Object Oriented Programming and Classes](../../../fundamentals/blob/master/fundamentals/oop_classes.md)<br>• [The `this` keyword](../../../fundamentals/blob/master/fundamentals/this.md) |[Reading Week 1](/Week1/README.md)|[Homework Week 1](/Week1/MAKEME.md)|
```
Comment (GeorgeSapkin) [inline README.md]: Why are these and other links relative? If it's in another repo use the full URL.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 >Please help us improve and share your feedback! If you find better tutorials or links, please share them by opening a Pull Request.
 
-# HackYourJavaScript
+# JavaScript3
```
Comment (GeorgeSapkin) [inline README.md]: Or maybe `HackYourFuture JavaScript 3` instead?

**Comment by remarcmij** [comment]
All links seem to work fine.

One thing that I sometimes found confusing is that the prereading for a lecture is to be found in the README/MAKEME of the previous lecture (hence the need for a Week 0). Can't we just have the prereading in the README of the corresponding lecture?

Content-wise, if I would teach this module again, I would consider moving OOP to week 3. This will leave more opportunity for practising async callbacks and promises in homework and will directly benefit the subsequent Node module. OOP / ES classes is a relatively stand-alone topic and isn't needed until later in the React module.

One thing missing in JS3 is probably error handling and try-catch.

I'm not clear on what we should teach for '(re)writing data structures (in JSON)'.

File: README.md (hunk 2 above)
Comment (mkruijt) [reply README.md]: yep I like it.

File: README.md (hunk 1 above)
Comment (mkruijt) [reply README.md]: see comment in other pr

**Comment by mkruijt** [comment]
"Can't we just have the prereading in the README of the corresponding lecture?" yes, we can do that I also have thought about removing the makeme entirely and puttting the homework and readings all in the readme. It could be that in some cases the file would become quite big.

For the content wise remarks, could you please create an issue, I rather solve one problem at a time :)

**Comment by mkruijt** [comment]
Oh and if i'm correct the error handeling is partly covered here: https://github.com/HackYourFuture/debugging, merging this repo with the javascript assignments is the next thing on my todo list 💃

**Comment by remarcmij** [comment]
`try` & `catch` and `new Error(...)` is not covered in the debugging repo. These topics are not about debugging. They are about writing code to handle runtime errors.

**Comment by remarcmij** [comment]
Can we finalize and merge this one, please? I would like to make some further changes as I need to teach this module to class 14.

**Pull request merged**